.github/workflows/continuous-integration.yml

```diff
@@ -87,10 +87,21 @@ jobs:
             nodeFlag: 18
             typescript: next
             typescriptFlag: next
-          # Node nightly
+          # Node 20
           - flavor: 5
-            node: 20-nightly
-            nodeFlag: 20_nightly
+            node: 20
+            nodeFlag: 20
+            typescript: latest
+            typescriptFlag: latest
+          - flavor: 6
+            node: 20
+            nodeFlag: 20
+            typescript: next
+            typescriptFlag: next
```

Comment:
We can remove this one; I'm trying to keep only a single entry for typescript@next.  I suspect that github actions throttle if you use it too much, so I try to keep the test matrix small where possible.